src/execution_plan/ops/shared/print_functions.c

```diff
@@ -17,7 +17,7 @@ static inline int _NodeToString(char *buf, uint buf_len, const char *alias, cons
 }
 
 int TraversalToString(const OpBase *op, char *buf, uint buf_len, AlgebraicExpression *ae) {
```

Comment:
Shouldn't this function return `uint` ?

Reply:
The porblem that this result is being later returned from ExpandIntoToString and this should be the signature of the func by redis:
typedef int (*fpToString)(const struct OpBase *, char *, uint);